website/docs/contribute/website.mdx

```diff
@@ -165,7 +165,7 @@ On Linux, you may find issues with the `node`, `npm`, or `yarn` version when you
 If so, re-install these packages in your venv with:
 ```bash
 nvm install node
-sudo apt install npm
+# sudo apt install npm
```

Comment:
```suggestion
```

Reply:
Gonna keep for now in case someone else needs it